typesafety/test_grpc.yml

```diff
@@ -222,3 +222,29 @@
     server = typing.cast(aio.Server, None)
     enable_server_reflection(["foo"], server, None)
 
+- case: issue_41_stream_multi_callable_type_unary
+  main: |
+    import grpc
+    import typing
+    def unary_stream() -> grpc.UnaryStreamMultiCallable[int, int]:
+      return typing.cast(grpc.UnaryStreamMultiCallable[int, int], None)
+
+    stream = unary_stream()
```

Comment:
This function returns the multi-callable object, not the stream. Call the multi-callable to get the stream.